hack/update-codegen.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
```

Comment:
Since most distros simply link sh to bash, no-one noticed when bash features slipped in to this script.